app/models/master_file.rb

```diff
@@ -723,6 +723,18 @@ def post_processing_file_management
     case Settings.master_file_management.strategy
     when 'delete'
       MasterFileManagementJobs::Delete.perform_now self.id
+    when 'move-ui-upload-only'
+      # check if the MasterFile.file_location contains the `dropbox_directory_name`
+      # If video uploaded via the UI file upload then video stored in temporary upload directory
+      # Move to a directory named after the collection
+      dropbox_directory_name = self.media_object.collection.dropbox_directory_name
```

Comment:
Not sure if I understand the question. 
- `Settings.master_file_management.path`: not sure where to put the files -- this setting can be specified
- `self.media_object.collection.dropbox_directory_name`: creating a collection generate a directory name from the collection name, I reusing this name. This might be overkill but aligns with how the `dropbox` path functions. 

I don't know if this is the best way.

Reply:
Ah, I misunderstood the `dropbox_directory_name`, thought it's another env var you needs to set up for all collections. 
I think this makes sense, creating a directory based on the collection name. What would happen if the directory doesn't exist? will the directory be created first? (Based on conversation with @jefferya this should be handled with existing "move" strategy)

Reply:
Correct. https://github.com/ualbertalib/avalon/blob/bugfix/file_disappear_447/app/jobs/master_file_management_jobs.rb#L44-L66